src/templates/layouts/healthCareLocalFacility/HealthServices/ServiceAddress.tsx

```diff
@@ -21,6 +17,10 @@ export const ServiceAddress = ({
   serviceLocationAddress,
   facilityAddress,
 }: ServiceAddressProps) => {
+  // TODO: When refactoring this, we won't ever need the `facilityAddress`, so
+  // we can remove it. If `field_use_facility_address` is true, we show no
+  // `addressData`.
+
```

Comment:
Yeah...fun fact...all this "which address should I use" logic is totally useless. 🙄 

It's either the `serviceLocationAddress` or it's nothing. We don't actually need `facilityAddress` in here at all. I have no idea why the Liquid template is like that.

Reply:
It feels like the liquid templates have a lot of cruft from old refactors and what-not in them. And sometimes just non-sensical for no reason